src/test/java/org/nuxeo/io/fsexporter/test/TestDriveLikeExporterPlugin.java

```diff
@@ -175,6 +175,23 @@ public void it_exports_only_the_main_blob_of_a_file() throws Exception {
 
     }
 
+    @Test
+    public void title_to_filename_handles_special_chars() throws Exception {
+        folder().at("/").withName("folder1").withTitle("Title of folder 1 on 10/29/2018").create(session);
```

Comment:
Please update the test to include all the possible chars in your forbidden pattern.

Reply:
good idea